lib/gemirro/cli/init.rb

```diff
@@ -27,9 +27,9 @@
     end
 
     # make sure index updates blank local specs
-    ['specs.4.8', 'latest_specs.4.8', 'prerelease_specs.4.8'].each do |s|
-      File.utime(Time.at(0), Time.at(0), File.join(directory, 'public', s))
-    end
+    # ['specs.4.8', 'latest_specs.4.8', 'prerelease_specs.4.8'].each do |s|
```

Comment:
No longer needed?

Reply:
Yeah, individual gemspec.rz are used, but all the marshaled stuff is depreciated / used for fallback